### PR TITLE
feat: Edit schema settings

### PR DESCRIPTION
**Pull request by connoratrug**
add, edit, delete works for schema settings

~~note: fix read only in forms components ( other pr #1336 )~~

### CODE REVIEW

**Comment by mswertz** [comment]
[(UMCG) Catalogue home page](https://trello.com/c/5XRlouYT/472-umcg-catalogue-home-page)

**Review by chinook25 — APPROVED** [review]
Works for me, just some minor comments.

File: apps/settings/src/components/SettingsManager.vue [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    },
+    handleRowDeleteRequest(setting) {
+      this.modalTitle = `Delete ${setting.key} setting`
+      this.settingActionLabel = "Delete Setting"
+      this.settingKey = setting.key
+      this.settingValue = setting.value
+      this.isKeyReadOnly = true,
+          this.isValueReadOnly = true,
+          this.actionFunction = this.deleteSetting
+      this.showModal = true
```
[the remark this answers is not in the document]
Comment (connoratrug) [reply apps/settings/src/components/SettingsManager.vue:104]: should be fixed by , https://github.com/molgenis/molgenis-emx2/pull/1336 , i'll check

File: apps/settings/src/components/SettingsManager.vue [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+      this.settings = resp._settings.filter(s => s.key !== "isOidcEnabled");
+    },
+    handleRowEditRequest(setting) {
+      this.modalTitle = `Edit ${setting.key} setting`
+      this.settingActionLabel = "Edit Setting"
+      this.settingKey = setting.key
+      this.settingValue = setting.value
+      this.isKeyReadOnly = true,
+          this.isValueReadOnly = false,
+          this.actionFunction = this.createSetting
```
[the remark this answers is not in the document]
Comment (connoratrug) [reply apps/settings/src/components/SettingsManager.vue]: done

File: apps/settings/src/components/SettingsManager.vue [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+  methods: {
+    async fetchSettings() {
+      const resp = await request("graphql", `{_settings{key, value}}`);
+      this.settings = resp._settings.filter(s => s.key !== "isOidcEnabled");
+    },
+    handleRowEditRequest(setting) {
+      this.modalTitle = `Edit ${setting.key} setting`
+      this.settingActionLabel = "Edit Setting"
+      this.settingKey = setting.key
+      this.settingValue = setting.value
```
[the remark this answers is not in the document]
Comment (connoratrug) [reply apps/settings/src/components/SettingsManager.vue]: done

File: apps/settings/src/components/SettingsManager.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (connoratrug) [reply apps/settings/src/components/SettingsManager.vue]: it's fixed , be CI preview broke

**Pull request merged**